docs/fast_data/configuration/config_maps/aggregation.md

```diff
@@ -310,6 +310,52 @@ Now that the `aliasOf` option is clear, we can have a look at the following conf
 
 As you can see, we used the same Projection twice, under different conditions: the first time we matched the record based on its identifier (`PEOPLE` dependency, without alias), the second time we matched the record based on the `MARRIAGE_b_TO_PEOPLE` condition (`PARTNER` dependency, with alias).
 
+:::tip
+`aliasOf` can also be used to rename long Projection names without the need of them being re-used. Instead of using `company_department_prefix_persons_table.name` we can alias the Projection to `persons` and use `persons.name` in the mapping! 
+:::
+
+Now let's imagine we need a `oneToMany` relationship, say, a `PEOPLE` to `CHILDREN` relationship. For that there's two things we need to consider. Let's see an example: 
+
+```json
+{
+   "version": "1.3.0",
+   "config": {
+      "SV_CONFIG": {
+         "dependencies": {
+            "PEOPLE": {
+               "type": "projection",
+               "on": "_identifier",
+            },
+            "CHILDREN": {
+               "type": "config",
+            },
+         },
+         "mapping": {
+            "name":"PEOPLE.name",
+            "children": "CHILDREN",
+         },
+      },
+      "CHILDREN": {
+         "joinDependency": "children",
+         "dependencies": {
+            "children": {
+               "type": "projection",
+               "on": "PEOPLE_TO_CHILDREN",
+               "aliasOf": "people",
+            },
+         },
+         "mapping": {
+            "name": "children.name",
+         },
+      },
+   },
+}
+```
+
+The first thing is that the `aliasOf` property always goes inside the `type: projection` configuration, as you can see we defined it in the `CHILDREN.dependencies.children` object instead of the `SV_CONFIG.dependencies.CHILDREN` object. The second is that we don't use `person` as a `joinDependency`, but we use the below aliased `children` instead.
```

Comment:
I don't understand the example. On top of the page, you are mentioning the alias persons.name but it's never used in the json. You are mentioning here that we are not using person as a joinDependency but I can't find it into the json (the aliasOf is people). Am I missing something?

Reply:
Ah you're right I've used persons instead of people. Althought if you see that part is a tip, it doesn't have much to do with what's down below. I mean it's related to the `aliasOf` but those are two separate cases. Would you rather some kind of divider more explicit maybe?
![image](https://github.com/mia-platform/documentation/assets/30315670/7b168287-c870-4428-a3b1-7d0e97092dfe)
That's how it looks like in web

Reply:
well, ok this is a separate tip, but IMHO it may be useful to use fields and words that are related to examples to make it clearer.

Anyway, we may add how it would be without using the aliasOf in comparison with how it is when using it

Reply:
no I think there is a misunderstanding here. Let me explain:
- tip box: it's ok to have it short like this, I was saying if it would be nice to use the same names of the examples that we are explaining. But since this is a different case just for long names of projections it's ok to leave it like this
- explanation below: I would like to have the 2 jsons, one without using aliasOf and the other one using it, so that I think it could be clearer

Reply:
Ok, I understand better, the thing though is that the part I added is not the explanation of the feature `aliasOf` itself. That part is before and as you suggested it comes with an example without the usage of `aliasOf` which makes a broken configuration. The part that I added explains how to use the `aliasOf` in case there's a `oneToMany` relationship so there's really no "wrong" usage of it.

Still I fell I'm missing something, maybe we can talk about it in the next meeting.